app.rb

```diff
@@ -1,20 +1,81 @@
+require_relative "models/apartment"
+require_relative "models/tenant"
 require_relative "data"
 apartments = data[:apartments]
 tenants = data[:tenants]
 
+
 # First, Open the data.rb an inspect the data. Identify and write, in comments, the following:
   # Explain how the data is structured
+  # Response: the data is arranged as an object that contains two arrays which each contain hashes which contain key value pairs
+
   # What are the properties for each of the two types of hashes
+  # Response: the properties for apartment array hash are: Id, address, monthly_rent and square
+  # Response: the properties for the tenants array hash are: Id, name, age, and apartment_id
 
 # Use enumerables to -
   # Print all the addresses for the apartments
+  #   apartments.each do |apartment|
+  #   puts apartment[:address]
+  # end
+
   # Print all the names for tenants
+      # tenants.each do |name|
```

Comment:
i'd use `tenant` instead of `name` for the iterator variable. `name[:name]` is confusing